migrations/800_groups__add.sql

```diff
@@ -0,0 +1,29 @@
+-- A Work in Progress to discuss shared schema for group storage in PL
+
+CREATE TABLE IF NOT EXISTS group_configs (
+    id BIGSERIAL PRIMARY KEY,
+    course_instance_id BIGINT NOT NULL REFERENCES course_instances(id) ON DELETE CASCADE ON UPDATE CASCADE,
+    assessment_id BIGINT REFERENCES assessments(id) ON DELETE CASCADE ON UPDATE CASCADE,
```

Comment:
Why restrict a config to a particular assessment? Especially if instructors want to, say, spread groups over multiple assessments for an end-of-semester ongoing project.

Reply:
The group developing that feature has a way to duplicate groups used on one assessment on others, which I think addresses that point. Database-wise, the `assessment_id` isn't required so there might be place for groups used across multiple assignments?